src/components/Settings/ChangesList.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import style from './style.css'
+
+export const ChangesList = ({ changes = [], translatedLabels = [] }) => {
```

Comment:
I'd rather see the only component in this file be the default export and drop the {} on the import

```suggestion
export default const ChangesList = ({ changes = [], translatedLabels = [] }) => {
```